tests/ts_simple_select_iso8601.erl

```diff
@@ -69,16 +69,19 @@
                 },
 
                 {0,
-                 {">", "2016-08-02 10:19"},
-                 {"<", "2016-08-02 10:20"}
-                },
-
-                {0,
                  {">", "2016-08-02 10:19.99"},
                  {"<", "2016-08-02 10:20"}
                 }
                ]).
 
+-define(FAIL_TESTS, [
+                {
+                  {1001,<<"The upper and lower boundaries are equal or adjacent. No results are possible.">>},
```

Comment:
Agree with @macintux here.  We should probably just match on the error number, not the text, since the text can change.